src/cascadia/CascadiaPackage/CascadiaPackage.wapproj

```diff
@@ -138,28 +138,26 @@
     </ItemGroup>
   </Target>
 
-  <!-- **BEGIN VC LIBS HACK** -->
   <!--
-    For our release builds, we're just going to integrate the UWPDesktop CRT into our package and delete the package dependencies.
-    It's very difficult for users who do not have access to the store to get our dependency packages, and we want to be robust
-    and deployable everywhere. Since these libraries can be redistributed, it's easiest if we simply redistribute them.
-    See also the "VC LIBS HACK" section in WindowsTerminal.vcxproj.
+    Some of our dependencies still require a CRT, so we're going to ship the forwarders in our package and

```

Comment:
File future work item to migrate the dependencies off?

Reply:
No future work planned - I suspect that XAML is not getting rid of its CRT dependency ;)